src/helpers/helpers.easing.js

```diff
@@ -1,194 +1,84 @@
 import {PI, TAU, HALF_PI} from './helpers.math';
 
+const atEdge = (t) => t === 0 || t === 1;
+const elasticIn = (t, s, p) => -(Math.pow(2, 10 * (t -= 1)) * Math.sin((t - s) * TAU / p));
+const elasticOut = (t, s, p) => Math.pow(2, -10 * t) * Math.sin((t - s) * TAU / p) + 1;
+// Math.pow(2, -10 * t) * Math.sin((t - 0.075) * TAU / 0.3) + 1,
 /**
  * Easing functions adapted from Robert Penner's easing equations.
  * @namespace Chart.helpers.easing.effects
  * @see http://www.robertpenner.com/easing/
  */
 const effects = {
-  linear(t) {
-    return t;
-  },
+  linear: t => t,
 
-  easeInQuad(t) {
-    return t * t;
-  },
+  easeInQuad: t => t * t,
 
-  easeOutQuad(t) {
-    return -t * (t - 2);
-  },
+  easeOutQuad: t => -t * (t - 2),
 
-  easeInOutQuad(t) {
-    if ((t /= 0.5) < 1) {
-      return 0.5 * t * t;
-    }
-    return -0.5 * ((--t) * (t - 2) - 1);
-  },
+  easeInOutQuad: t => ((t /= 0.5) < 1)
+    ? 0.5 * t * t
+    : -0.5 * ((--t) * (t - 2) - 1),
 
-  easeInCubic(t) {
-    return t * t * t;
-  },
+  easeInCubic: t => t * t * t,
 
-  easeOutCubic(t) {
-    return (t -= 1) * t * t + 1;
-  },
+  easeOutCubic: t => (t -= 1) * t * t + 1,
 
-  easeInOutCubic(t) {
-    if ((t /= 0.5) < 1) {
-      return 0.5 * t * t * t;
-    }
-    return 0.5 * ((t -= 2) * t * t + 2);
-  },
+  easeInOutCubic: t => ((t /= 0.5) < 1)
+    ? 0.5 * t * t * t
+    : 0.5 * ((t -= 2) * t * t + 2),
 
-  easeInQuart(t) {
-    return t * t * t * t;
-  },
+  easeInQuart: t => t * t * t * t,
```

Comment:
wouldn't `t ^ 4`  be better for ease of readability?
Same goes for some of the other ones

Reply:
I'd think its that way for speed, but not sure.

Reply:
Did some quick googling and testing and this seems indeed to be the case, its about 1.5 to 2% faster, think the performance gain outweighs the readability gain since at max it is multiplied 5 times